x/emissions/module/rewards/scores.go

```diff
@@ -129,7 +129,7 @@ func GenerateReputerScores(
 	}
 
 	// Update topic quantile of EMA score
-	topicEmaScoreQuantile, err := actorutils.GetQuantileOfScores(emaScores, topic.ActiveReputerQuantile)
+	topicEmaScoreQuantile, err := actorutils.GetQuantileOfScores(newScores, topic.ActiveReputerQuantile)
```

Comment:
variable name is wrong assuming you intend to use instantaneous scores over ema scores